### ISSUE TITLE
devices disconnect to unmanned rocket

### ISSUE THREAD

**Issue by Petrisis** [zmaster587/AdvancedRocketry#1335]
## Version of Advanced Rocketry
AdvancedRocketry-1.12.2-1.4.0-84-universal
## Version of LibVulpes
LibVulpes-1.12.2-0.2.8-32-universal
## Version of Minecraft
1.12.2

## Description of the problem
Having a space station above a gas giant and two unmanned assemblers with two working rockets (doesnt matter that there two rockets). Fueling station, monitoring station and fluid unloader are situated in the same chunk (and chunkloaded) together with rocket and rocket assembly. fueling, monitoring and unloading blocks are connected to assembly - not directly to the rocket! 
a redstone circuit controlles the monitoring station.
this setup works perfectly on first launch. upon return, none of this blocks remember the rocket or the rocket assembly... no way to automate this for long

**Comment by ct-klein** [comment]
I can confirm that this is still happening in AdvancedRocketry-1.12.2-1.4.1-114-universal, LibVulpes-1.12.2-0.3.0-44-universal.
I have a single unmanned rocket, fueling station, monitoring station and fluid unloader.  I do not have any redstone in place for automation.  All 3 are linked to the assembly control.  It launches fine the first time, comes back.  At this point the linker that I used to connect the devices returns to saying 'Coords unset'.

The only way I have found to resolve the issue is to 'disassemble' the unmanned ship and go thru the scan/build/relink process.
